src/components/common/FileUploadPdf.vue

```diff
@@ -32,12 +32,6 @@ export default class FileUploadPdf extends Vue {
   @Prop({ default: null }) readonly pageSize!: PageSizes
   @Prop({ required: true }) readonly userId!: string
 
-  // Network service functions
-  @Prop({ required: true })
-  readonly getPresignedUrl!: (fileName: string) => Promise<PresignedUrlIF>
```

Comment:
For some reason, when passing this function as a prop, it seems to be unable to read the Legal Services getter that provides the URL. So, I just reference the class.function below.